src/core_ocean/get_cvmix.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
 ## CVMix Tag for build
-CVMIX_TAG=v0.84-beta
+CVMIX_TAG=lt_v1.3
 ## Subdirectory in CVMix repo to use
 CVMIX_SUBDIR=src/shared
 
 ## Available protocols for acquiring CVMix source code
-CVMIX_GIT_HTTP_ADDRESS=https://github.com/CVMix/CVMix-src.git
-CVMIX_GIT_SSH_ADDRESS=git@github.com:CVMix/CVMix-src.git
-CVMIX_SVN_ADDRESS=https://github.com/CVMix/CVMix-src/tags
-CVMIX_WEB_ADDRESS=https://github.com/CVMix/CVMix-src/archive
+CVMIX_GIT_HTTP_ADDRESS=https://github.com/qingli411/CVMix-src.git
+CVMIX_GIT_SSH_ADDRESS=git@github.com:qingli411/CVMix-src.git
+CVMIX_SVN_ADDRESS=https://github.com/qingli411/CVMix-src/tags
+CVMIX_WEB_ADDRESS=https://github.com/qingli411/CVMix-src/archive
```

Comment:
We can't use @qingli's repo here, as then all mainline E3SM simulations would depend on a nonstandard CVMix tag.  Worse, we need to be able to go back to any previous E3SM/master commit and run it in future years.  That means all submodules also need to be permanent.

The solution is to simply not make the above changes.  We can still merge this PR.  In order to run with @qingli411's non-default option, one would need to make the above change with your local repo copy.  

After Qing's changes are merged into CVMIX, we make an MPAS PR to update the above `CVMIX_TAG` line.